uc_attribute/views/uc_attribute_handler_field_order_product_attribute.inc

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Field handler.
+ *
+ * Provides a human-readable version of
+ * the selected combination of attributes.
+ */
+class UcAttributeHandlerFieldOrderProductAttribute extends views_handler_field {
+
+  /**
+   * Defines a few default options for the combination field.
+   */
+  public function option_definition() {
+    $options = parent::option_definition();
+
+    return $options;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function render($values) {
+    $data = unserialize($values->{$this->field_alias});
+    $result = "";
+    if (is_array($data['attributes'])) {
```

Comment:
This throws `Notice: Undefined index: attributes in UcAttributeHandlerFieldOrderProductAttribute->render() (line 25 ` when you have some products without attributes.

Suggests:

```php
if (!empty($data['attributes'])) {
```

I believe `$data['attributes']` will always be an array if it's not empty.